convert_matrices.py

```diff
@@ -78,12 +78,16 @@ def to_mtx(self, input_dir: Path, output_dir: Path):
 @dataclass(frozen=True)
 class CSVPerCell:
     directory: str
-    name: str = 'cell_files'
+    cohort: Optional[str] = None
     path_filter: Optional[Callable[[Path], bool]] = None
     entry_filter: Optional[RowFilter] = None
     sep: str = ','
     expr_column: int = 1
 
+    @property
+    def name(self) -> str:
+        return self.directory if self.cohort is None else '__'.join([self.directory, self.cohort])
```

Comment:
```suggestion
        return self.directory if self.cohort is None else '/'.join([self.directory, self.cohort])
```

 should work, too, and keeps the translation logic contained in `matrix_dir()`. We should also add an assert to `matrix_dir()` to make sure that `__` doesn't occur in the argument. 

Also, surely `a + '/' + b` is faster (and simpler) than `'/'.join([a, b])`